flixel/FlxCamera.hx

```diff
@@ -526,119 +529,128 @@ class FlxCamera extends FlxBasic
-		#end
-		drawItem.addQuad(frame, matrix, cr, cg, cb, ca);
-	}
-	
-	public function copyPixels(?frame:FlxFrame, ?pixels:BitmapData, ?sourceRect:Rectangle, destPoint:Point, cr:Float = 1.0, cg:Float = 1.0, cb:Float = 1.0, ca:Float = 1.0, blend:BlendMode = null, smoothing:Bool = false):Void
-	{
-		_helperMatrix.identity();
-		_helperMatrix.translate(destPoint.x + frame.offset.x, destPoint.y + frame.offset.y);
-		var isColored:Bool = (cr != 1.0) || (cg != 1.0) || (cb != 1.0) || (ca != 1.0);
-		#if !FLX_RENDER_TRIANGLE
-		var drawItem:FlxDrawTilesItem = startQuadBatch(frame.parent, isColored, blend, smoothing);
-		#else
-		var drawItem:FlxDrawTrianglesItem = startTrianglesBatch(frame.parent, smoothing, isColored, blend);
-		#end
-		drawItem.addQuad(frame, _helperMatrix, cr, cg, cb, ca);
-	}
-	
-	public function drawTriangles(graphic:FlxGraphic, vertices:DrawData<Float>, indices:DrawData<Int>, uvtData:DrawData<Float>, colors:DrawData<Int> = null, position:FlxPoint = null, blend:BlendMode = null, smoothing:Bool = false):Void
-	{
-		_bounds.set(0, 0, width, height);
-		var isColored:Bool = (colors != null && colors.length != 0);
-		var drawItem:FlxDrawTrianglesItem = startTrianglesBatch(graphic, smoothing, isColored, blend);
-		drawItem.addTriangles(vertices, indices, uvtData, colors, position, _bounds);
-	}
-#else
-	public function drawPixels(?frame:FlxFrame, ?pixels:BitmapData, matrix:Matrix, cr:Float = 1.0, cg:Float = 1.0, cb:Float = 1.0, ca:Float = 1.0, blend:BlendMode = null, smoothing:Bool = false):Void
-	{
-		buffer.draw(pixels, matrix, null, blend, null, (smoothing || antialiasing));
+		if (FlxG.renderBlit)
+		{
+			buffer.draw(pixels, matrix, null, blend, null, (smoothing || antialiasing));
+		}
+		else
+		{
+			var isColored:Bool = (cr != 1.0) || (cg != 1.0) || (cb != 1.0) || (ca != 1.0);
+			#if !FLX_RENDER_TRIANGLE
+			var drawItem:FlxDrawTilesItem = startQuadBatch(frame.parent, isColored, blend, smoothing);
+			#else
+			var drawItem:FlxDrawTrianglesItem = startTrianglesBatch(frame.parent, smoothing, isColored, blend);
+			#end
+			drawItem.addQuad(frame, matrix, cr, cg, cb, ca);
+		}
 	}
 	
 	public function copyPixels(?frame:FlxFrame, ?pixels:BitmapData, ?sourceRect:Rectangle, destPoint:Point, cr:Float = 1.0, cg:Float = 1.0, cb:Float = 1.0, ca:Float = 1.0, blend:BlendMode = null, smoothing:Bool = false):Void
 	{
-		if (pixels != null)
+		if (FlxG.renderBlit)
 		{
-			buffer.copyPixels(pixels, sourceRect, destPoint, null, null, true);
+			if (pixels != null)
+			{
+				buffer.copyPixels(pixels, sourceRect, destPoint, null, null, true);
+			}
+			else if (frame != null)
+			{
+				frame.paint(buffer, destPoint, true);
+			}
 		}
-		else if (frame != null)
+		else
 		{
-			frame.paint(buffer, destPoint, true);
+			_helperMatrix.identity();
+			_helperMatrix.translate(destPoint.x + frame.offset.x, destPoint.y + frame.offset.y);
+			var isColored:Bool = (cr != 1.0) || (cg != 1.0) || (cb != 1.0) || (ca != 1.0);
+			#if !FLX_RENDER_TRIANGLE
```

Comment:
I decided to refactor one thing at a time. FLX_RENDER_BLIT and FLX_RENDER_TILE moving to FlxG.renderBlit & FlxG.renderTile is a big enough change already, I didn't want to break existing functionality and figured any refactoring for render triangle could wait for a future PR.